test/utils/objects/objects.go

```diff
@@ -54,7 +54,7 @@ func TestNROScheduler() *nropv1.NUMAResourcesScheduler {
 			Name: objectnames.DefaultNUMAResourcesSchedulerCrName,
 		},
 		Spec: nropv1.NUMAResourcesSchedulerSpec{
-			SchedulerImage: "quay.io/openshift-kni/scheduler-plugins:4.17-snapshot",
+			SchedulerImage: "quay.io/openshift-kni/scheduler-plugins:4.18-snapshot",
```

Comment:
because of the scheduler references, we need to branch out and fix the scheduler (operand) image first, then the operator